src/Mod/OpenSCAD/OpenSCADCommands.py

```diff
@@ -387,7 +395,42 @@ def addelement(self):
 
         except OpenSCADUtils.OpenSCADError as e:
             FreeCAD.Console.PrintError(e.value)
-
+     
+    def refreshelement(self):
+        print('Refresh Element')
+        doc=FreeCAD.activeDocument()
+        if doc :
+           for obj in doc.Objects :
+               doc.removeObject(obj.Name)
+        self.addelement()
+
+    def loadelement(self):
+        print('Load Element')
+        print('Load Element')
+        filename, filter = QtGui.QFileDialog.getOpenFileName(parent=self.form, caption='Open file', dir='.', filter='OpenSCAD Files (*.scad)',selectedFilter='',option=0)
+
+        if filename:
+           print('filename :'+filename)
+           fp = open(filename,'r')
+           with fp:
+                data = fp.read()
+                self.form.textEdit.setText(data)
+    
+    def saveelement(self) :
+        print('Save Element')
+        filename, filter = QtGui.QFileDialog.getSaveFileName(parent=self.form, caption='Open file', dir='.', filter='OpenSCAD Files (*.scad)',selectedFilter='',option=0)
+
+        if filename:
+           print('filename :'+filename)
+           Text = self.form.textEdit.toPlainText()
+           fp = open(filename,'w')
+           with fp:
+              fp.write(Text)
+    
+class OpenSCADMeshBooleanWidget(QtGui.QWidget):
+    def __init__(self,*args):
+        QtGui.QWidget.__init__(self,*args)
+        #self.textEdit=QtGui.QTextEdit()
```

Comment:
These lines are duplicated with above ones. I even don't know how it can pass CI.